addons/transform/org.openhab.transform.feed.test/FeedTransformationServiceTest.launch

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I know what it is and was in doubt whether it should be included. @kaikreuzer what do you think is it a good habbit to add them?